main.py

```diff
@@ -1,27 +1,36 @@
-import PySimpleGUI as sg      
+import PySimpleGUI as sg    
+from json import dumps 
 
 sg.theme("LightGray1")
-print("Distância")
-Distância=int(input("Qual foi a quilometragem?"))
-Metros=(Distância)
-print('Tempo')
-Tempo=int(input("Qual foi o seu tempo?"))
-Velocidade=(Tempo/Distância)
-print("Essa foi a sua velocidade:")
-print(Velocidade," Minutos por quilometro")
+# print("Distância")
+# Distância=int(input("Qual foi a quilometragem?"))
+# Metros=(Distância)
+# print('Tempo')
+# Tempo=int(input("Qual foi o seu tempo?"))
+# Velocidade=(Tempo/Distância)
+# print("Essa foi a sua velocidade:")
+# print(Velocidade," Minutos por quilometro")
 
-layout = [[sg.Text("Qual foi a quilometragem?")], [sg.InputText()]] 
-
-layout = [[sg.Text("Qual foi seu tempo?")], [sg.InputText()], [sg.Submit(), sg.Cancel()]]
+layout = [[sg.Text("Qual foi a distância percorrida?")], [sg.InputText(key='km'), sg.Combo(['km','m'], 'km', key='un_dist')],
+          [sg.Text("Qual foi seu tempo?")], [sg.InputText(key='tempo'), sg.Combo(['hora(s)','minuto(s)'], 'minuto(s)', key='un_tempo')], 
+          [sg.Submit(), sg.Cancel()]]
 
 window = sg.Window("Recordes", layout)    
 
 while True:
     event, values = window.read()
 
-    if event == sg.WIN_CLOSED or event == "Quit": break
-    
-    text_input = values[0]
-    sg.popup("You entered", text_input)
+    # if event == sg.WIN_CLOSED or event == "Quit": break
```

Comment:
Aqui o exemplo usando o operador **in**, é o equivalente matemático a 'pertence a' (**∈**) .